libqtile/backend/wayland/cffi/build.py

```diff
@@ -188,11 +190,23 @@ def get_include_path(lib: str) -> str:
     f.write(f"TARGET := {LIBWAYC}\n")
 
 
-def build_library() -> None:
+def hash_headers(files: list[str]) -> str:
+    h = hashlib.sha256()
+    for file in files:
+        path = QW_PATH / file
+        with open(path, "rb") as f:
+            h.update(f.read())
+    return h.hexdigest()
+
+
+def build_library(clean: bool = False) -> None:
+    if clean:
+        subprocess.run(["make", "-C", QW_PATH / ".." / "cffi", "clean"], check=True)
+
     subprocess.run(["make", "-C", QW_PATH / ".." / "cffi"], check=True)
 
 
-def ffi_compile(verbose: bool = False) -> None:
+def ffi_compile(verbose: bool = False, force: bool = False) -> None:
```

Comment:
`--force` here is weird to me, and suggests something else about the dependency calculations is incorrect. Can we just fix that instead?